app/screens/reader-screen/reader-screen.tsx

```diff
@@ -27,6 +29,24 @@ export class ReaderScreen extends React.Component<Props> {
     this.list.current.props.onRefresh()
     this.props.readerStore.fetchCreatorList()
     this.props.readerStore.fetchBookmarkList()
+
+    // Automatically switch to following list if the user has followees
+    if (this.props.navigation.state.routeName !== "Following") {
+      reaction(
+        () => this.props.readerStore.hasFetchedCreatorList,
+        (hasFetchedCreatorList, r) => {
+          if (hasFetchedCreatorList) {
+            r.dispose()
+            if (
+              this.props.readerStore.followingCreators.length > 0 &&
+              this.props.navigation.state.routeName !== "Following"
```

Comment:
In case `hasFetchedCreatorList` triggers too slow, and user have already navigated away from reader screen, this might cause strange UX?
Maybe want to check if use is still in the reader screen too

Reply:
@williamchong007 Changed to trigger only in featured list